CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [FSDP]: limited support of shared weights between FSDP wrappers. This allows large parameter
           and gradient memory to be sharded despite being needed from different layers due to
           weight sharing. [#836]
+- SlowMoDistributedDataParallel[feature][experimental] ([#378])
```

Comment:
This entry should have some short description, like the other ones (also remove the parentheses for consistency)

Reply:
Thanks for the comment. The reason I kept it without a description and with parentheses was following along the example of past instances when a feature is added. I noticed short descriptions in the cases where some feature was modified and not added. Could you please take a look at those and let me know your thoughts on whether we should a short description?

Reply:
Looking more closely at this file, it looks like there are indeed inconsistencies in the formatting over time, however as far as I can tell the large majority of entries have some description, and all recent ones use `[#XXX]` without parentheses.
@blefaudeux may know better though